test/encoding.ts

```diff
@@ -26,6 +26,11 @@ export interface Trade {
   owner: string;
 }
 
+export interface Interaction {
+  target: string;
+  callData: string | Bytes;
```

Comment:
nit: I would use the `ethers` type for "something that can be coerced into bytes" directly (they export it)
```suggestion
  callData: BytesLike;
```

Note this requires

```ts
import { BigNumber, BytesLike } from "ethers";
```

Reply:
@bh2smith You forgot me!

Reply:
Resolved in #241